src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -199,21 +199,47 @@ export const populateNotificationsWithRedirectLinksAndStatusColors = (
 };
 
 export const activateTribunalOrdersAndRequestsLink = (items: SendNotificationTypeItem[], req: AppRequest): void => {
```

Comment:
Is it actually required to pass the whole AppRequest here? can see we are only using req.session?.userCase

Reply:
Now passing userCase instead.